### ISSUE TITLE
Unify property names for Inventory and InventoryItem

### ISSUE THREAD

**Issue by Nordsoft91** [peter-kish/gloot#207]
Currently item protoset property name is inconsistent:
In **Inventory** class name is **item_protoset**
In **InventoryItem** class name is **protoset**

Suggest to unify name

**Comment by peter-kish** [comment]
Yup, the naming is inconsistent. Sadly, it will break compatibility with previous versions if fixed, so it will have to be implemented in the next major release (in fact it's already implemented on the [dev_v3.0.0](https://github.com/peter-kish/gloot/tree/dev_v3.0.0) branch).

**Comment by peter-kish** [comment]
Since version 3.0.0 the property names are aligned
